external/easylogging++/easylogging++.cc

```diff
@@ -1248,7 +1248,8 @@ bool OS::termSupportsColor(void) {
   std::string term = getEnvironmentVariable("TERM", "");
   return term == "xterm" || term == "xterm-color" || term == "xterm-256color"
          || term == "screen" || term == "linux" || term == "cygwin"
-         || term == "screen-256color" || term == "screen.xterm-256color";
+         || term == "screen-256color" || term == "screen.xterm-256color"
```

Comment:
Are there any terminals ending in "-256color" which don't support color? Might make sense to just future-proof it rather than constantly whitelisting new terminals, either way, someone can always use the escape hatch of setting TERM.

Reply:
i agree !

i thought about just checking for "-256color" but in a way it's just as hacky, there's plenty of terminals that support color but don't have that in the name, but do have their own terminfo.

The official way would be to query the terminfo database eg `tigetnum("colors") >= 256` if 256 colors are wanted. But this needs a library like ncurses or termcap, not sure that's a current dependency.

Reply:
The only relevant concern would be whether one needs a whitelist because there are terminals ending in "-256color" which don't support color; I doubt there are. Using a terminfo database for something so trivial (colorized logging) seems unwarranted.

Reply:
> The only relevant concern would be whether one needs a whitelist because there are terminals ending in "-256color" which don't support color; I doubt there are.

Makes sense, will update it to check for `-color` or `-256color` instead.